storage/teststore/queue_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package teststore
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/storage"
+)
+
+func TestQueue(t *testing.T) {
```

Comment:
Create a test in `testsuite` so it can be shared by redis and teststore.

Reply:
moved